api/src/main/java/io/opentelemetry/baggage/ImmutableBaggage.java

```diff
@@ -22,24 +17,29 @@
 @Immutable
 // TODO: Migrate to AutoValue
 // @AutoValue
-class BaggageSdk implements Baggage {
+public class ImmutableBaggage implements Baggage {
```

Comment:
Does this need to be public?

Reply:
hmm. good point. I have to adjust a silly test, but that's easy enough to do.

Reply:
fixed